features/solr/pom.xml

```diff
@@ -93,6 +93,36 @@
             <artifactId>admin-core-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-transports-http</artifactId>
+            <version>3.4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.24.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-hdfs-client</artifactId>
+            <version>3.2.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.calcite</groupId>
+            <artifactId>calcite-core</artifactId>
+            <version>1.26.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-webapp</artifactId>
+            <version>9.4.33.v20201020</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-server</artifactId>
+            <version>9.4.33.v20201020</version>
```

Comment:
❗ It looks like this will conflict with the version of Jetty that Pax Web brings in.

https://github.com/codice/ddf/blob/master/pom.xml#L225

Reply:
Can I upgrade the version that Pax Web brings in too? 
That version is vulnerable via: https://snyk.io/vuln/SNYK-JAVA-ORGECLIPSEJETTY-575561

Reply:
upgraded Pax Web jetty version too, no errors during profile:install or in the logs

Reply:
Removed this change altogether, updating the pax web jetty version caused test failures